sklearn/metrics/tests/test_common.py

```diff
@@ -360,7 +360,13 @@
 # No Sample weight support
 METRICS_WITHOUT_SAMPLE_WEIGHT = [
     "cohen_kappa_score",
-    "confusion_matrix",
+    "confusion_matrix", # Left this one here because the tests in this file do
+                        # not work for confusion_matrix, as its output is a
+                        # matrix instead of a number. Testing of
+                        # confusion_matrix with sample_weight is in
+                        # test_classification.py
+    "hamming_loss",
+    "matthews_corrcoef_score",
```

Comment:
you can remove the "hamming loss" and "matthews_corrocef_score". They were supported recently